### PR TITLE
sci-libs/bliss: New upstream release

### PR DESCRIPTION
**Pull request by kiwifb**
Package-Manager: Portage-3.0.28, Repoman-3.0.3
Signed-off-by: François René Pierre Bissey <frp.bissey@gmail.com>

### CODE REVIEW

File: sci-libs/bliss/bliss-0.77.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (AndrewAmmerlaan) [inline sci-libs/bliss/bliss-0.77.ebuild:4]: Can we bump this to `EAPI=8` please?

File: sci-libs/bliss/bliss-0.77.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+HOMEPAGE="https://users.aalto.fi/~tjunttil/bliss/index.html"
+
+LICENSE="LGPL-3"
+SLOT="0/1"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="gmp"
+
+RDEPEND="gmp? ( dev-libs/gmp:0= )"
+DEPEND="${RDEPEND}
+	app-arch/unzip"
```
Comment (AndrewAmmerlaan) [inline sci-libs/bliss/bliss-0.77.ebuild:19]: unzip is a `BDEPEND`

File: sci-libs/bliss/bliss-0.77.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+PATCHES=( "${FILESDIR}/${PN}-0.77-install.patch" )
+
+# No tests are currently provided upstream
+#RESTRICT="test"
```
Comment (AndrewAmmerlaan) [inline sci-libs/bliss/bliss-0.77.ebuild:23]: Just remove these comments, no need to restrict the tests if there are no tests anyway

**Review by AndrewAmmerlaan — CHANGES_REQUESTED** [review]

File: sci-libs/bliss/bliss-0.77.ebuild (hunk 3 above)
Comment (kiwifb) [reply sci-libs/bliss/bliss-0.77.ebuild:23]: That, I was supposed to remove before the push. Got distracted.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**